### PR TITLE
925962-Update the UG documentation for scheduler control newly implemented features.

### PR DESCRIPTION
**Pull request by sanjayvikramsf4373**
### Description: ###

I have added the UG content for below feature:

- Number of Visible Weeks support - WinUI Scheduler.

### CODE REVIEW

File: winui/Scheduler/Month-View.md [hunk 1]
```diff
@@ -260,6 +260,24 @@ this.Schedule.MonthViewSettings.TrailingDaysVisibility = Visibility.Collapsed;
 
 ![customize-leading-and-trailing-days-visibility-in-winui-scheduler-month-view](Month-View_Imges/customize-leading-and-trailing-days-visibility-in-winui-scheduler.png)
 
+## Customizing the Number of Visible Weeks
```
Comment (Karthickmani97) [inline winui/Scheduler/Month-View.md:263]: Change - Customize number of month rows

File: winui/Scheduler/Month-View.md [hunk 2]
```diff
@@ -260,6 +260,24 @@ this.Schedule.MonthViewSettings.TrailingDaysVisibility = Visibility.Collapsed;
...
+The number of week visible in the month view can be changed by setting the NumberOfVisibleWeeks property of the MonthViewSettings in SfScheduler. The default value is 6, and the valid range is from 1 to 6. Even if leading and trailing days are hidden, they will appear when changing the number of visible weeks, as no additional days exist during this change.
```
Comment (Karthickmani97) [inline winui/Scheduler/Month-View.md:264]: The number of week visible in the month view can be changed by setting the `NumberOfVisibleWeeks` property of `MonthViewSettings` in `SfScheduler`.  By default, the number of visible weeks is 6.

File: winui/Scheduler/Month-View.md (hunk 1 above)
Comment (Karthickmani97) [inline winui/Scheduler/Month-View.md:263]: Move to below [Appointment display count](https://help.syncfusion.com/winui/scheduler/month-view#appointment-display-count)

File: winui/Scheduler/Month-View.md [hunk 3]
```diff
@@ -260,6 +260,24 @@ this.Schedule.MonthViewSettings.TrailingDaysVisibility = Visibility.Collapsed;
...
+
+{% tabs %}
+{% highlight xaml %}
```
Comment (Karthickmani97) [inline winui/Scheduler/Month-View.md:267]: Follow below format

image

File: winui/Scheduler/Month-View.md [hunk 4]
```diff
@@ -260,6 +260,24 @@ this.Schedule.MonthViewSettings.TrailingDaysVisibility = Visibility.Collapsed;
...
+<scheduler:SfScheduler x:Name="Schedule" 
+                     ViewType="Month">
+    <scheduler:SfScheduler.MonthViewSettings>
+        <scheduler:MonthViewSetting NumberOfVisibleWeeks="3"/>
```
Comment (Karthickmani97) [inline winui/Scheduler/Month-View.md:271]: Highlight this line

File: winui/Scheduler/Month-View.md [hunk 5]
```diff
@@ -260,6 +260,24 @@ this.Schedule.MonthViewSettings.TrailingDaysVisibility = Visibility.Collapsed;
...
+    </scheduler:SfScheduler.MonthViewSettings>
+</scheduler:SfScheduler>
+{% endhighlight %}
+{% highlight c#%}
+this.Schedule.ViewType = SchedulerViewType.Month;
+this.Schedule.MonthViewSettings.NumberOfVisibleWeeks = 3;
```
Comment (Karthickmani97) [inline winui/Scheduler/Month-View.md:277]: Highlight this line

File: winui/Scheduler/Month-View.md [hunk 6]
```diff
@@ -260,6 +260,24 @@ this.Schedule.MonthViewSettings.TrailingDaysVisibility = Visibility.Collapsed;
...
+{% endhighlight %}
+{% endtabs %}
+
```
Comment (Karthickmani97) [inline winui/Scheduler/Month-View.md:280]: Add notes:

- The week number range is limited to values between 1 and 6. Any value outside this range will cause the `NumberOfVisibleWeeks` to default to 6.
- The `NumberOfVisibleWeeks` property dynamically controls the number of weeks displayed in the scheduler's month view.
- The `DisplayDate` property defines the first row of dates shown in the scheduler's month view.

**Review by Karthickmani97 — COMMENTED** [review]
@sanjayvikramsf4373 , address review correction

File: winui/Scheduler/Month-View.md (hunk 4 above)
Comment (sanjayvikramsf4373) [reply winui/Scheduler/Month-View.md:274]: highlighted the line

File: winui/Scheduler/Month-View.md [hunk 7]
```diff
@@ -260,6 +260,24 @@ this.Schedule.MonthViewSettings.TrailingDaysVisibility = Visibility.Collapsed;
...
+{% tabs %}
+{% highlight xaml %}
+<scheduler:SfScheduler x:Name="Schedule" 
+                     ViewType="Month">
+    <scheduler:SfScheduler.MonthViewSettings>
+        <scheduler:MonthViewSetting NumberOfVisibleWeeks="3"/>
+    </scheduler:SfScheduler.MonthViewSettings>
+</scheduler:SfScheduler>
+{% endhighlight %}
```
[the remark this answers is not in the document]
Comment (sanjayvikramsf4373) [reply winui/Scheduler/Month-View.md:278]: added line.

File: winui/Scheduler/Month-View.md (hunk 5 above)
Comment (sanjayvikramsf4373) [reply winui/Scheduler/Month-View.md:283]: Highlighted the line.

File: winui/Scheduler/Month-View.md (hunk 6 above)
Comment (sanjayvikramsf4373) [reply winui/Scheduler/Month-View.md:288]: Added notes.

File: winui/Scheduler/Month-View.md [hunk 8]
```diff
@@ -260,6 +260,35 @@ this.Schedule.MonthViewSettings.TrailingDaysVisibility = Visibility.Collapsed;
 
 ![customize-leading-and-trailing-days-visibility-in-winui-scheduler-month-view](Month-View_Imges/customize-leading-and-trailing-days-visibility-in-winui-scheduler.png)
 
+## Customize the number of visible weeks
```
Comment (Jeyasri-Murugan) [inline winui/Scheduler/Month-View.md:263]: Change the number of visible weeks

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**